src/mmw/apps/modeling/migrations/0044_project_weather_simulations.py

```diff
@@ -0,0 +1,80 @@
+future_weather_stations = ['13739','13781','14734','14735','14737','14751','14777','14778','4725','93730','94728']
+common_weather_stations = list(set(recent_weather_stations) & set(future_weather_stations))
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('modeling', '0043_remove_sedaadjust_from_tr55_scenarios'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='project',
+            name='weather_simulations',
+            field=models.JSONField(default=list, help_text='JSON array of simulation groups available for project'),
+        ),
+        # First set all GWLFE projects to have neither enabled (the deafult value)
+        migrations.RunSQL(f'''
+            UPDATE modeling_project
+            SET weather_simulations = '{WEATHER_SIMULATIONS["neither"]}'::jsonb
+            WHERE model_package = 'gwlfe';
+        '''),
+        # Then set those eligible for recent weather
+        migrations.RunSQL(f'''
+            UPDATE modeling_project
+            SET weather_simulations = '{WEATHER_SIMULATIONS["recent"]}'::jsonb
+            WHERE model_package = 'gwlfe'
+              AND (
+	            SELECT COUNT(station_id)
+	            FROM (
+		            SELECT (station_data ->> 'station')::int AS station_id
+		            FROM jsonb_array_elements(gis_data::jsonb -> 'WeatherStations') as station_data
+                ) AS stations_in_project
+	            WHERE station_id = ANY(ARRAY[{",".join(recent_weather_stations)}])
+              ) = 2;
+        '''),
+        # Then set those eligible for future weather
+        migrations.RunSQL(f'''
+            UPDATE modeling_project
+            SET weather_simulations = '{WEATHER_SIMULATIONS["future"]}'::jsonb
+            WHERE model_package = 'gwlfe'
+              AND (
+	            SELECT COUNT(station_id)
+	            FROM (
+		            SELECT (station_data ->> 'station')::int AS station_id
+		            FROM jsonb_array_elements(gis_data::jsonb -> 'WeatherStations') as station_data
+                ) AS stations_in_project
+	            WHERE station_id = ANY(ARRAY[{",".join(future_weather_stations)}])
+              ) = 2;
+        '''),
+        # Finally set those eligible for both
+        migrations.RunSQL(f'''
+            UPDATE modeling_project
+            SET weather_simulations = '{WEATHER_SIMULATIONS["both"]}'::jsonb
+            WHERE model_package = 'gwlfe'
+              AND (
+	            SELECT COUNT(station_id)
+	            FROM (
+		            SELECT (station_data ->> 'station')::int AS station_id
+		            FROM jsonb_array_elements(gis_data::jsonb -> 'WeatherStations') as station_data
+                ) AS stations_in_project
+	            WHERE station_id = ANY(ARRAY[{",".join(common_weather_stations)}])
+              ) = 2;
```

Comment:
Should probably explain this magic query more. `jsonb_array_elements` converts an array of JSON elements into individual rows. When we `COUNT(station_id)` that are in the given array, and total to 2, that means both weather stations are in that array, which is what we need to test before assigning a simulation to a project.